docs/expectation_gallery/azure-pipelines-expectation-gallery.yml

```diff
@@ -75,24 +75,126 @@ stages:
                 [ScriptsChanged]
                 assets/scripts/**
 
-  - stage: deploy_gallery_staging
+  - stage: exp_tests_on_all_backends
     dependsOn: scope_check
     condition: or(eq(variables.isScheduled, true), eq(variables.isRelease, true), eq(variables.isManual, true))
     pool:
       vmImage: 'ubuntu-20.04'
 
     jobs:
-      - job: build_gallery
-        timeoutInMinutes: 420
+      - job: pandas
+        timeoutInMinutes: 180
+        variables:
+          python.version: '3.8'
+
+        steps:
+          - task: UsePythonVersion@0
+            inputs:
+              versionSpec: '$(python.version)'
+            displayName: 'Use Python $(python.version)'
+
+          - bash: python -m pip install --upgrade pip
+            displayName: 'Update pip'
+
+          - script: |
+              pip install \
+                "cryptography==38.0.4" \
```

Comment:
Do we need this pin (for this and all other jobs)? Hopefully we can remove it.

Reply:
Dunno, it was needed in all the pipelines several months ago because of some issue.

Reply:
We shouldn't pin this here. If there are expectations that require a specific version of a library that should be defined in the expectation itself. Pinning here would give us a false sense of security that the expectation would work, but most likely the user's environment would have the latest version of `cryptography`. The expectation should fail the test and need to be updated. Also pinning this specific package that is intimately related to security is something we should avoid. 

As an aside - and not relevant in this case for expectation tests but worth mentioning - in general, if there is a specific version that is causing an issue, we should use the `!=` modifier to avoid that particular version but not future versions.

Reply:
It was for a now resolved snowflake issue https://github.com/great-expectations/great_expectations/pull/6755

Removed.